src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -77,7 +77,9 @@ public function __construct(ResourceNameCollectionFactoryInterface $resourceName
     public function __invoke(array $context = []): OpenApi
     {
         $baseUrl = $context[self::BASE_URL] ?? '/';
-        $info = new Model\Info($this->openApiOptions->getTitle(), $this->openApiOptions->getVersion(), trim($this->openApiOptions->getDescription()));
+        $contact = new Model\Contact($this->openApiOptions->getContactName(), $this->openApiOptions->getContactUrl(), $this->openApiOptions->getContactEmail());
+        $license = null === $this->openApiOptions->getLicenseName() ? null : new Model\License($this->openApiOptions->getLicenseName(), $this->openApiOptions->getLicenseUrl());
+        $info = new Model\Info($this->openApiOptions->getTitle(), $this->openApiOptions->getVersion(), trim($this->openApiOptions->getDescription()), $this->openApiOptions->getTermsOfService(), $contact, $license);
```

Comment:
Why `trim`?

Reply:
The trim() was added by you. I didnt want to remove it.

Reply:
I checked that part and dont see why it would be needed. Shall i remove it within this PR ?

Reply:
oh you're right I had an issue with a trailing newline, please keep that sorry for the comment.